### PR TITLE
Readme updates

### PR DESCRIPTION
**Pull request by thecoderpanda**
Updated readme for prompt questions after running the `multi_clone.py` file.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,3 +44,18 @@ pip install -r requirements.txt
 # run the setup
 python multi_clone.py
 ```
+When you execute `python multi_clone.py`, you will see following prompts that guide you through setting up your node(s).
```
Comment (anomit) [inline README.md:47]: add screenshots or screen recordings

File: README.md [hunk 2]
```diff
@@ -44,3 +44,18 @@ pip install -r requirements.txt
...
+
+1. **Terminate Existing containers:** "Do you want to kill all running containers and screen sessions of testnet nodes? (y/n) n"
+
+- Type `y` Use this option and stop all active containers or node instances. This will clean up all the older containers. Please cross check your running containers before executing this command. 
+
+2. **Custom Slot ID Deployment:** "Do you want to deploy a custom index of slot IDs (indices begin at 0, enter in the format [begin, end])? (indices/n)"
```
Comment (anomit) [inline README.md:53]: elaborate on the case when `n` is entered, where the entire array of slot IDs are chosen to deploy nodes against

File: README.md (hunk 1 above)
Comment (thecoderpanda) [reply README.md:47]: Noted. Working on it

File: README.md (hunk 2 above)
Comment (thecoderpanda) [reply README.md:53]: Oh yes! Apologies. Missed that case. Will add and push another commit!

Comment (thecoderpanda) [reply README.md:55]: Addressed

File: README.md (hunk 1 above)
Comment (thecoderpanda) [reply README.md:47]: Addressed

**Pull request merged**